### ISSUE TITLE
[Bug]: Local API Error (Click to copy): TypeError: Cannot read properties of undefined (reading 'content')

### ISSUE THREAD

**Issue by ThiloteE** [FreeTubeApp/FreeTube#3592]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads)
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Click on "Trending" tab
2. No videos show up in "default tab"
3. Following Error messages trigger: 
   ![image](https://github.com/FreeTubeApp/FreeTube/assets/73715071/1cce630f-9fcc-4c13-8717-dc03aee48eef)

4. Within "Trending", click the "music", "gaming" or "films" tab and videos will show up, but not preview.
   ![image](https://github.com/FreeTubeApp/FreeTube/assets/73715071/3393b14b-bcfb-48a8-ac69-82315851dddc)

### Expected Behavior

"Trending" and "Most popular" content should load properly.

### Issue Labels

API issue, content not loading

### FreeTube Version

v0.18.0 Beta

### Operating System Version

Windows 10 Pro; Version 22H2; Build: 19045.2965; Windows Feature Experience Pack 1000.19041.1000.0

### Installation Method

.exe

### Primary API used

Invidious API

### Additional Information

Here are my current general settings:

![image](https://github.com/FreeTubeApp/FreeTube/assets/73715071/0ebb9346-56f6-4476-ad94-5e1a9d8c4809)

I have tried a few different Invidious instances, but for all of them the behaviour is similar.
I don't remember exactly when this problem first emerged, as it has been a while and I only now was bothered enough to report it.

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by ThiloteE** [comment]
Ah, preview of videos is shown if "local API" is set as "preferred API backend", but "default" tab is empty regardless.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@ThiloteE ur invidious instance is typed wrong

Remove everything and start with https:// 

It will show u a list, select the instance u like from the list

**Comment by ThiloteE** [comment]
Ohhhhh. You are totaly right!

My bad. I don't know exactly how it came to be. "Trending" was not working and while trying to fix it, i visited https://api.invidious.io/ and just copied the name of an instance from there. Maybe a few months ago I tried to set a new instance and made the same mistake? I don't know.

Feature request: If a link is explicitly required for instances to work, do not allow users to enter anything other than a link.

Closing this, as your comment made me find the culprit. Thank you very much :-)
